tests/unit/models/test_evc_deploy.py

```diff
@@ -186,16 +187,20 @@ def test_prepare_flow_mod(self):
         expected_flow_mod = {
             "match": {"in_port": interface_a.port_number},
             "cookie": evc.get_cookie(),
+            "owner": "mef_eline",
             "actions": [
                 {"action_type": "output", "port": interface_z.port_number}
             ],
             "priority": EVPL_SB_PRIORITY,
+            "table_group": "EVPL",
+            "table_id": 0,
         }
         self.assertEqual(expected_flow_mod, flow_mod)
 
     def test_prepare_pop_flow(self):
         """Test prepare pop flow  method."""
         attributes = {
+            "table_group": {"base": 0},
```

Comment:
Let's keep the test case as closely as possible with its actual table group that will be used, can you update this?

Reply:
Changed this one too.

Reply:
Looks good.